frameworks/helloworld/src/main/dist/shm.yml

```diff
@@ -0,0 +1,22 @@
+name: {{FRAMEWORK_NAME}}
+scheduler:
+  principal: {{FRAMEWORK_PRINCIPAL}}
+  user: {{FRAMEWORK_USER}}
+pods:
+  hello:
+    count: {{HELLO_COUNT}}
+    ipc-mode: PRIVATE
+    shm-size: 128
+    tasks:
+      server:
+        shm-size: 256
```

Comment:
Probably we should validate this field at framework level - If the `ipc-mode` is SHARE_PARENT, uses are not allowed to specify the `shm-size` field, because we should be explicit otherwise users may think the 256 mb shm may be used somewhere.

Based on the [API](https://github.com/apache/mesos/blob/master/include/mesos/mesos.proto#L3322~L3329), no matter this field is being ignored or errored out, I would suggest frameworks always add this validation, so that users know their config error before launching the task.

Sorry for the previous example, we have already changed the unit tests correspondingly. Mesos will return a failure if shm-size is specified in SHARE_PARENT mode. Please update here. :)